docs/README.md

```diff
@@ -11,7 +11,7 @@
 NGXS is a state management pattern + library for Angular. It acts as a single source of
 truth for your application's state, providing simple rules for predictable state mutations.
 
-NGXS is modeled after the CQRS pattern popularly implemented in libraries like Redux and NGRX
+NGXS is modeled after the CQRS pattern popularly implemented in libraries like Redux and Ngrx
```

Comment:
`NgRx` then

Reply:
Changes every non-link occurrence of "Ngrx" to "NgRx", thanks for pointing that out.